Dockerfile-slim.template

```diff
@@ -18,9 +18,9 @@ RUN set -ex; \
 # uid                  PostgreSQL Debian Repository
 	key='B97B0AFCAA1A47F044F244A07FCC7D46ACCC4CF8'; \
 	export GNUPGHOME="$(mktemp -d)"; \
-	(gpg --keyserver ha.pool.sks-keyservers.net --recv-keys "$key" \
-	    || gpg --keyserver ipv4.pool.sks-keyservers.net --recv-keys "$key"); \
-	gpg --export "$key" > /etc/apt/trusted.gpg.d/postgres.gpg; \
+	(gpg --batch --keyserver ha.pool.sks-keyservers.net --recv-keys "$key" \
```

Comment:
This is good. 

I think it would be valuable to include why the maintainers of the Docker Hub PostgreSQL container image are using `--batch` as a fix, and why it works.

If you see [this issue](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=913614#27) on the Debian bug tracker, they state:

> For any script use you should anyway use --batch which disables the use
of the tty as a side-effect.

>PS i do encourage everyone who is automating the use of gpg to use
--batch everywhere, as this forces GnuPG into a mode that is expected to
be used for automation (its "API", for lack of a better term, as opposed
to its "UI", which is its normal non-batch mode).  And, FWIW, i agree
with Tianon that GnuPG should simply assume --batch if no tty exists,
but that's not the kind of change i can fit into debian stable, i think.

For future reference, Tianon is one of the lead maintainers of https://github.com/docker-library.